src/caselawclient/models/documents/body.py

```diff
@@ -144,6 +144,13 @@ def has_content(self) -> bool:
 
         return False
 
+    @cached_property
+    def has_external_data(self) -> bool:
+        """Is there data which is not present within the source document:
+        is there a spreadsheet which has populated some fields. The current implementation
+        "is there a uk:party tag" is intended as a stopgap whilst we're not importing that data."""
+        return bool(self._xml.xml_as_tree.xpath("//uk:party", namespaces=DEFAULT_NAMESPACES))
```

Comment:
is this the best proxy we have? If so we will have to be very careful anytime change is made to the parser to make sure this is still the case. I want to spend a little bit more time thinking if we have an alternative